auraed/src/init/system_runtimes/pid1_system_runtime.rs

```diff
@@ -104,6 +108,8 @@ impl SystemRuntime for Pid1SystemRuntime {
         let socket_addr = socket_address
             .unwrap_or_else(|| DEFAULT_NETWORK_SOCKET_ADDR.into())
             .parse::<SocketAddr>()?;
-        create_tcp_socket_stream(socket_addr).await
+        let stream = create_tcp_socket_stream(socket_addr).await?;
```

Comment:
isn't this the same?  `await` and `?` then wrap in `Ok` seems the same as just returning the results of the `await` directly.

Reply:
Removed in https://github.com/aurae-runtime/aurae/pull/506/commits/9a35b2fc94984777d84b99156fff80194f434b05